### PR TITLE
Add partial B lifetime contributions from dbcu and sbcu operators

### PR DESCRIPTION
**Pull request by smeiser**

### CODE REVIEW

File: eos/b-decays/lifetime_TEST.cc [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+                    Options oo
+                    {
+                        { "model", "CKM" },
+                        { "q",     "d"   }
+                    };
+
+                    Lifetime d(p, oo);
+
+                    // compare to known decay width of
+                    // (1.519 +/- 0.004 ps)^-1
```
Comment (dvandyk) [inline eos/b-decays/lifetime_TEST.cc:93]: Can you please add the value and uncertainty in units of inverse picoseconds, just like in line 76?

File: eos/b-decays/lifetime_TEST.cc [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+                    Options oo
+                    {
+                        { "model", "CKM" },
+                        { "q",     "s"   }
+                    };
+
+                    Lifetime d(p, oo);
+
+                    // compare to known decay width of
+                    // (1.521 +/- 0.005 ps)^-1
```
Comment (dvandyk) [inline eos/b-decays/lifetime_TEST.cc:110]: See above.

**Review by dvandyk — COMMENTED** [review]
General approval. Two nitpicking comments/requests in the test case.

File: eos/b-decays/observables.cc [hunk 3]
```diff
@@ -2152,6 +2153,43 @@ namespace eos
...
+                        &Lifetime::decay_width_dbcu,
+                        std::make_tuple(),
+                        { { "q", "u"} }),
+
+                /* B_s^0 lifetime */
+                make_observable("B_s^0::Gamma(dbcu)", R"(\Gamma(\bar{B}_s^0)^{dbcu})",
+                        Unit::InversePicoSecond(),
+                        &Lifetime::decay_width_dbcu,
+                        std::make_tuple(),
+                        { { "q", "s"} })
```
Comment (dvandyk) [inline eos/b-decays/observables.cc:2185]: As discussed just now: ``sbcu`` sector observables are still missing.

**Comment by dvandyk** [comment]
@smeiser Is this ready to be merged. @mreboud and I cannot see any further issues.

**Comment by smeiser** [comment]
@dvandyk I think it should be fine as it is and can be merged

**Pull request merged**